### DISCUSSION TITLE
The ASP.NET OTEL (OpenTelemetry) Metrics dashboard doesn't work with using of the lates version of nuget packages.

### DISCUSSION THREAD

**Discussion by askrinnik** [nazarii-piontko/sample-dotnet-otel#1] [General]
Hello,

Thank you so much for the excellent demo system and the Grafana Dashboard with metrics.
I created a new application and wanted to use the dashboard. But it turned out that it doesn't show values.
I figured out that your application uses the **"OpenTelemetry.Exporter.Prometheus.AspNetCore" Version="1.4.0-rc.1"**. This version of the package publishes instruments with names like `process_cpu_time_s`. 

But the package was updated in August with the following PR: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4753. It changes names to something like `process_cpu_time_seconds_total`. So, the dashboard doesn't work with new names.

I'd really appreciate it if you had a chance to update your application with the latest OTEL Nuget packages and improve the dashboard.

P.S. I use a different architecture solution in my demo application. It pushes metrics to OTEL Collector.  Then, Prometheus scrapes values from the collector. It looks a bit flexible because it is unnecessary to configure Prometheus to scrape metrics from **all** application instances. it is enough to scrape from the collector.
You can find my example [here](https://github.com/askrinnik/OtlpMeterUsing/tree/master/ConsolePushApp).
The docker-compose file for such approach is [here](https://github.com/askrinnik/OtlpMeterUsing/tree/master/dependencies/Meter-Push).

My example is based on [the following example](https://github.com/karlospn/opentelemetry-metrics-demo). 
The collector publishes metrics with names like `process_cpu_time`. So, the example above has own dashboard which works with the names.

Best regards

**Comment by nazarii-piontko** [comment]
Thank you a lot for pointing to this issue. I really appreciate it. Unfortunately I didn't follow these updates for awhile. I will update project as soon as possible.

**Reply by askrinnik** [reply]
Thank you so much for the updated code and the dashboard.
It would be great to update the example code on the [Grafana dashboard page](https://grafana.com/grafana/dashboards/17706-asp-net-otel-metrics/).

**Comment by nazarii-piontko** [comment]
Fixed.
